src/cli/parse-flags.ts

```diff
@@ -320,6 +314,14 @@ const setCLIArg = (flags: ConfigFlags, rawArg: string, normalizedArg: string, va
     } else {
       throwCLIParsingError(rawArg, 'expected to receive a valid log level but received nothing');
     }
+  } else {
+    // we haven't found this flag in any of our lists of arguments, so we
+    // should put it in our list of unknown arguments
+    flags.unknownArgs.push(rawArg);
+
+    if (typeof value === 'string') {
+      flags.unknownArgs.push(value);
+    }
```

Comment:
Do we prevent duplicate args either up or down stream? Or do we just not care and let the respective CLI handle that (i.e. Jest can filter those out)?